components/Dialogs/examples/DialogsDismissingExampleViewController.m

```diff
@@ -176,7 +178,9 @@ + (NSDictionary *)catalogMetadata {
 
 @interface ProgrammaticViewController ()
 
-@property(nonatomic, strong) MDCFlatButton *dismissButton;
+@property(nonatomic, strong) MDCButton *dismissButton;
+
+@property(nonatomic, strong) MDCContainerScheme *containerScheme;
```

Comment:
I think to support the Dragons app we may need to initialize the container scheme in each of the examples. This can be done as a follow-up.